portfolio/src/main/webapp/src/components/theme-switcher.ts

```diff
@@ -0,0 +1,39 @@
+import * as Surplus from 'surplus';
+import S from 's-js';
+
+import '@style/index.dark.scss';
+import '@style/index.light.scss';
+import {html} from '@src/util/html';
+
+type Theme = 'light' | 'dark';
+
+const currentTheme = S.data('light' as Theme);
+
+/**
+ * Initializes the theme switcher.
+ */
+export function activate() {
+  S(() => {
```

Comment:
Rely on vanilla JS instead of using third party libraries. This predictably changes so you can just update the attribute you care about directly.

Reply:
Fixed in #10